### PR TITLE
Davidoesch patch 1

### PR DESCRIPTION
**Pull request by danduk82**

### CODE REVIEW

File: chsdi/static/doc/source/services/sdiservices.rst [hunk 1]
```diff
@@ -1039,6 +1039,55 @@ Example
...
+URL
+***
+
+- vector tiles: https://vectortiles.geo.admin.ch/mbtiles/
+- gl-styles: https://vectortiles.geo.admin.ch/gl-styles/
+
+Metadata Service
+****************
+
+Each tileset has a corresponding metatda `json` file that describes the available set of tiles.
```
Comment (oterral) [inline chsdi/static/doc/source/services/sdiservices.rst]: missing 'a'

**Comment by loicgasser** [comment]
We should give examples with om and mapboxgl.

**Comment by davidoesch** [comment]
we will provide it in www.geo.admin.ch produkt page including isntructions  to how to use it in QGIS

**Comment by davidoesch** [comment]
This One 

We will provide data links in CMS

**Comment by davidoesch** [comment]
hi @danduk82 can't find it on todays deploy. since we would like to go live soon: when will this be available on api3.geo.admin.ch?

**Comment by danduk82** [comment]
@davidoesch , when we decide for an offical go-live for the service vectortiles. Anyway after the service is stable on S3.

**Comment by davidoesch** [comment]
well the idea is, as metioend last monday, to be ready to define the go live next monday, but for this the description needs to be online.

no worry if it is already online silent... we just loose time if we wait.

**Pull request closed without merging**